packages/flutter/test/widgets/text_test.dart

```diff
@@ -414,7 +414,7 @@ void main() {
             TestSemantics(
               label: 'INTERRUPTION',
               textDirection: TextDirection.rtl,
-              rect: const Rect.fromLTRB(448.0, 0.0, 488.0, 80.0),
+              rect: const Rect.fromLTRB(0.0, 0.0, 40.0, 80.0),
```

Comment:
@GaryQian is this expected?

Reply:
This may actually be a bug, but I'll look at that separately. I think in practice the rects still show up in the right place after transforms, but it may be better to make sure the values are rolled into here too. I think we can go ahead with this for now in the context of your semantics fix and ill fix this when I take a deeper look.

Reply:
If we don't fix this here, we should at least file an issue and link it from here.

Reply:
https://github.com/flutter/flutter/issues/34433

Reply:
I believe I have a fix for this that I'll be landing separately.  I have a patch ready to go that will be changing this again, but don't want to fiddle with it or risk having yet another piece to worry about as this patch is blcoking the google3 roll now.